### PR TITLE
export keccak function as interface

### PR DESCRIPTION
**Pull request by huazhouwang**
## What does this implement/fix? Explain your changes.
因为 iOS 没有 keccak 哈希方法，需要向上层暴露 keccak @CivelXu 

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
…

## Pull request type
 
_Put an `x` in the boxes that apply_
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

File: electrum_gui/android/console.py [hunk 1]
```diff
@@ -2242,6 +2242,14 @@ def dapp_eth_send_tx(self, tx_hex: str):
     def dapp_eth_rpc_info(self):
         return json.dumps(PyWalib.get_rpc_info())
 
+    def dapp_eth_keccak(self, message: str):
```
Comment (taimanhui) [inline electrum_gui/android/console.py:2245]: 返回值是固定的类型的话有需要用下type hint吗？

**Review by taimanhui — APPROVED** [review]

File: electrum_gui/android/console.py (hunk 1 above)
Comment (huazhouwang) [reply electrum_gui/android/console.py:2245]: ack

File: electrum_gui/android/console.py [hunk 2]
```diff
@@ -2242,6 +2242,14 @@ def dapp_eth_send_tx(self, tx_hex: str):
     def dapp_eth_rpc_info(self):
         return json.dumps(PyWalib.get_rpc_info())
 
+    def dapp_eth_keccak(self, message: str) -> str:
+        if message.startswith("0x"):
+            message_bytes = bytes.fromhex(message[2:])
```
Comment (shuaiLiWang) [inline electrum_gui/android/console.py:2247]: message[2:]是否可以替换为eth_utils.remove_0x_prefix(message)?

**Review by shuaiLiWang — APPROVED** [review]

File: electrum_gui/android/console.py (hunk 2 above)
Comment (huazhouwang) [reply electrum_gui/android/console.py:2247]: ack

**Pull request merged**